### DISCUSSION TITLE
How to save the debug or point cloud from dbscan in csv format?

### DISCUSSION THREAD

**Discussion by vimalchawda** [isl-org/Open3D#3629] [Q&A] [answered]
http://www.open3d.org/docs/release/tutorial/geometry/pointcloud.html#DBSCAN-clustering

**Comment by ssheorey** [comment, marked as answer]
That tutorial converts the obtained cluster labels to a Numpy array. You can then save the Numpy array as csv with `np.savetext`. Something like:

`np.savetxt('pcd_labels.csv', np.hstack((np.asarray(pcd.points), labels)), delimiter=',')`
